### ISSUE TITLE
Don't allow to save GPX layer into database until Layman supports it

### ISSUE THREAD

**Issue by jmacura** [hslayers/hslayers-ng#3900]
# Bug

**Describe the bug**
We allow user to select "save to database" even for GPX layers, which is not yet supported AFAIK.

**To Reproduce**
1. Go to 'Datasources' -> 'File' -> 'GPX'
2. Select 'Save to database': 'yes'
3. Click '+ Add'
4. See error in console and layer upload stuck

**Expected behavior**
Not possible to select 'Save to database' (shall be hidden or greyed out).
Alternatively, implement GPX support in Layman.

**Screenshots**
![image](https://user-images.githubusercontent.com/5598693/224435663-51db3bfe-9d32-48a0-bfb3-7772e7b69966.png)
![image](https://user-images.githubusercontent.com/5598693/224435742-cfbbd800-8f18-429d-aab8-14a15b692bc6.png)

**Additional context**
related discussion #3330

**Comment by FilipLeitner** [comment]
GPX features are converted into JSON so it should be all right from this perspective. 
I tested with one GPX file and  it was saved and loaded as expected [see here ](https://watlas.lesprojekt.cz/rest/workspaces/filip/layers/sample)

Can you maybe provide the file you were testing with?

**Comment by jmacura** [comment]
> GPX features are converted into JSON so it should be all right from this perspective.

Hmm, that was also my impression until I tried... 🤔 So there must be something special about the GPX file or something wrong in my workflow...

> Can you maybe provide the file you were testing with?

Sure, the file I was trying to add can be downloaded from here: https://drive.google.com/file/d/1OnYjJejaG_D_3iKCI4ZwqqJTD__drySq/view

**Comment by FilipLeitner** [comment]
Uploaded  on [issue-3899](https://watlas.lesprojekt.cz/rest/workspaces/filip/layers/skryse2023_test_filip?0.9877717737895199) branch (there is a small fix regarding SLD upload which was affecting GPX as well but no connection to the feature upload pipeline so it should be ok) and also on [develope](https://watlas.lesprojekt.cz/rest/workspaces/filip/layers/skryse2023_develop_filip?0.7853849407708523=) (just to be sure) without any issue as well

**Comment by jmacura** [comment]
> Uploaded on [issue-3899](https://watlas.lesprojekt.cz/rest/workspaces/filip/layers/skryse2023_test_filip?0.9877717737895199) branch (there is a small fix regarding SLD upload which was affecting GPX , among others, but no connection to the feature upload pipeline so it should be ok) and also on [develop](https://watlas.lesprojekt.cz/rest/workspaces/filip/layers/skryse2023_develop_filip?0.7853849407708523=) (just to be sure) without any issue as well

OK, I confirm uploading from localhost works fine. But I fail to upload data from both https://hub4everybody.com/map/ and https://watlas.lesprojekt.cz/ . Layman always hangs up with 404 error.
![image](https://user-images.githubusercontent.com/5598693/225097648-b73f84d2-51b4-46c7-8e48-1f3a2bae20da.png)
